document/js-api/index.bs

```diff
@@ -767,7 +774,15 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
         1. Let |ret| be ? [=Call=](|func|, undefined, |jsArguments|). If an exception is thrown, trigger a WebAssembly trap, and propagate the exception to the enclosing JavaScript.
         1. Let [|parameters|] → [|results|] be |functype|.
         1. If |results| is empty, return undefined.
-        1. Otherwise, return [=ToWebAssemblyValue=](|ret|, |results|[0]).
+        1. Otherwise, if |results| contains a single element, return [=ToWebAssemblyValue=](|ret|, |results|[0]).
+        1. Otherwise,
+            1. Let |method| be ? [=GetMethod=](|results|, @@iterator).
+            1. If |method| is undefined, [=throw=] a {{TypeError}}.
+            1. Let |values| be ? [=IterableToList=](|ret|, |method|).
+            1. Let |wasmValues| be a new, empty [=list=].
+            1. For each |value| and |resultType| in |values| and |results|,
```

Comment:
Nit: To be pedantic, this wording is ambiguous, since it could either describe a join or a cross product.

Also, maybe add an assert that both have the same arity?

Reply:
Good point. I'm not sure what wording is the best; what do you think of this?

Reply:
LGTM